docs/manual/kinds/sros.md

```diff
@@ -0,0 +1,649 @@
+    - endpoints: ["sr-sim-dist-iom-1:1/2/c1/1", "sr-sim-dist-iom3:3/1/c1/1"]     #(3)!
+```
+
+Inside the SR OS container, these interfaces are mapped so they follow Linux interface name conventions: `eL-xX-M-cC-P`, `eL-xX-M-P`, `eL-M-cC-P`, or `eL-M-P`. Note that the prefix `e` is added at the beginning of the port, and the forward slash `/` is replaced with a hyphen `-`. Some practical examples are shown below.
+
+/// admonition
+    type: Port_Naming
+Nokia SR-SIM port mapping examples
+```
+e1-2-3       -> card 1, mda 2, port 3
+e1-2-c3-1    -> card 1, mda 2, connector 3, port 1
+e2-2-c3-4    -> card 2, mda 2, connector 3, port 4
+e1-x2-3-4    -> card 1, xiom 2, mda 3, port 4
+e1-x2-3-c4-5 -> card 1, xiom 2, mda 3, connector 4, port 5
+```
+///
+
+The example above is the equivalent to the topology file below:
+
+```yaml
+  links:
+    - endpoints: ["sr-sim1:e1-1-c1-1", "sr-sim2:e1-1-1"]                           #(1)!
+    - endpoints: ["sr-sim-dist-iom-1:e1-1-c1-1", "sr-sim-dist-iom-2:e2-x1-1-c1-1"] #(2)!
+    - endpoints: ["sr-sim-dist-iom-1:e1-2-c1-1", "sr-sim-dist-iom3:e3-1-c1-1"]     #(3)!
+
+```
+
+1. sr-sim1 port 1/1/c1/1 on linecard 1 is connected to sr-sim2 port 1/1/1 on linecard 1
+
+2. sr-sim port 1/1/c1/1 on linecard 1 is connected to sr-sim port 2/x1/1/c1/1 on linecard 2
+
+3. sr-sim port 1/2/c1/1 on linecard 1, MDA 2 is connected to sr-sim port 3/1/c1/1 on linecard 3, MDA 1
+
+
+The management interface for the SR-SIM is typically mapped to `eth0` of the Linux namespace where the container is running. 
+
+Interfaces of an integrated system are defined with an endpoint to the container node as usual.
+
+Distributed systems require certain settings given the nature of the SR-SIM simulator:
+  
+  1. Containers must all run in the same Linux namespace. This is currently achieved using the `network-mode` directive in clab[^1].
+  2. The containers sharing namespace are all bridged internally to an internally created switch, which is simply a Linux bridge with uniquely named interfaces. Users do not need to configure the switch unless they have a specific need to use the `NOKIA_SROS_FABRIC_IF` environment variable  to override the default interfaces [^2]. 
+  3. Datapath links for the SR-SIM node SHOULD[^3] be connected to the container emulating the specific linecard.
+
+Example topologies for Integrated and Distributed nodes are shown below:
+
+/// tab | Integrated SR-SIM
+```yaml
+name: "sros"
+mgmt:
+  network: srsim_mgmt
+  ipv4-subnet: 10.78.140.0/24
+topology:
+  kinds:
+    nokia_srsim:
+      license: /opt/nokia/sros/license.txt
+      image: nokia_srsim:25.7.R1
+  nodes:
+    sr-sim10:
+      kind: nokia_srsim
+      type: SR-1 # Implicit default
+    sr-sim11: 
+      kind: nokia_srsim
+  links:
+    # Datapath Interfaces
+    - endpoints: ["sr-sim10:e1-1-c1-1", "sr-sim11:e1-1-c1-1"]    
+    - endpoints: ["sr-sim10:e1-1-c1-2", "sr-sim11:e1-1-c1-2"]    
+```
+///
+
+/// tab | Distributed SR-SIM
```

Comment:
Do we want to have this variant (nodes per linecard) promoted in the docs?

If so, should it really also be the first distributed option to list?